praetorian_cli/handlers/module.py

```
@@ -0,0 +1,424 @@
+    click.echo(f"  Repository:  {mod.get('repo', '')}")
+    click.echo(f"  Installed:   {installed_str}")
+    click.echo(f"  Target:      {mod.get('target_type', 'asset')}")
+    click.echo(f"  Description: {mod.get('description', '')}")
+
+    tags = mod.get("tags", [])
+    if tags:
+        click.echo(f"  Tags:        {', '.join(tags)}")
+
+    options = mod.get("options", {})
+    if options:
+        click.echo(f"\n  Options:")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)